AEPCore/Mocks/TestableExtensionRuntime.swift

```diff
@@ -37,8 +39,9 @@ public class TestableExtensionRuntime: ExtensionRuntime {
         dispatchedEvents += [event]
     }
 
-    public func createSharedState(data: [String: Any], event _: Event?) {
+    public func createSharedState(data: [String : Any], xdmData: [String : Any]?, event: Event?) {
```

Comment:
if we are providing an API like this we should make both data and xdmData non-optionals.